app/assets/javascripts/application/nav-close.js

```diff
@@ -0,0 +1,11 @@
+$(document).click(function(event) {
+  $navmenu = $('#nav-menu');
+  $target = $(event.target);
+  if(!$target.closest('#nav-menu').length &&
+  $('#nav-menu').is(":visible")) {
+    $('#nav-menu').css('display', 'none');
+  }
+  else {
+      $('#nav-menu').css('display', 'inline-block');
+  }
```

Comment:
This file would be easier to read with more conventional whitespace, eg:
```
if (!$target.closest('#nav-menu').length && $('#nav-menu').is(":visible")) {
  $('#nav-menu').css('display', 'none');
} else {
  $('#nav-menu').css('display', 'inline-block');
}
 ```
For javascript, we should use single quotes consistently for strings. In the medium term, we should add a javascript linter to the action center so we can catch these kinds of things automatically.

Reply:
Oh, this is some old code. I'm not actually using this, I'm only using css. I'll delete it.